test/test_va_api_fixture.cpp

```diff
@@ -189,6 +240,55 @@ void VAAPIFixture::doFillConfigAttribList()
     EXPECT_EQ(m_configAttribList.size(), m_vaConfigAttribs.size());
 }
 
+void VAAPIFixture::doTestCreateImage(uint32_t currentFmt,
+		std::pair<uint32_t, uint32_t> currentResolution)
+{
+    VAStatus status;
+    VAImageFormat currentImageFormat;
+
+    currentImageFormat.fourcc = currentFmt;
+
+    status = vaCreateImage(m_vaDisplay, &currentImageFormat,
+		 currentResolution.first, currentResolution.second, &m_image);
+
+    if (status == VA_STATUS_SUCCESS)
+	ASSERT_ID(m_image.image_id);
+    else
+	EXPECT_TRUE(
+		status == VA_STATUS_ERROR_OPERATION_FAILED);
+}
+
+bool VAAPIFixture::doFindFormat (VAImageFormat imageFormat, uint32_t current)
```

Comment:
VAImageFormat should be a const reference

Reply:
Updated.